diff --git a/dist/index.js b/dist/index.js
index dd8e55d55dd900fd86235652446c00f36cbf3896..8ea7ea70d790f762eb0b6588d170489ae09c6c24 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -324,7 +324,7 @@ function Nav() {
     /* @__PURE__ */ jsxs6("div", { className: "nx-flex nx-grow nx-flex-wrap nx-items-center nx-justify-end nx-gap-3", children: [
       navPages.map((page) => {
         var _a2;
-        const name = ((_a2 = page.frontMatter) == null ? void 0 : _a2.title) || page.name;
+        const name = (page.frontMatter && page.frontMatter.menuTitle) || ((_a2 = page.frontMatter) == null ? void 0 : _a2.title) || page.name;
         if (page.active) {
           return /* @__PURE__ */ jsx6(
             "span",
@@ -380,7 +380,7 @@ function PostsLayout({
     return /* @__PURE__ */ jsxs8("div", { className: "post-item", children: [
       /* @__PURE__ */ jsx8("h3", { children: /* @__PURE__ */ jsx8(Link4, { href: post.route, passHref: true, legacyBehavior: true, children: /* @__PURE__ */ jsx8("a", { className: "!nx-no-underline", children: postTitle }) }) }),
       description && /* @__PURE__ */ jsxs8("p", { className: "nx-mb-2 dark:nx-text-gray-400 nx-text-gray-600", children: [
-        description,
+        jsx8(MDXTheme, { children: [description] }),
         config.readMore && /* @__PURE__ */ jsx8(Link4, { href: post.route, passHref: true, legacyBehavior: true, children: /* @__PURE__ */ jsx8("a", { className: "post-item-more nx-ml-2", children: config.readMore }) })
       ] }),
       date && /* @__PURE__ */ jsx8(
